packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -160,6 +160,9 @@ export interface IDetailsListProps extends React.Props<DetailsList> {
 
   /** Boolean value to indicate if the role application should be applied on details list. Set to false by default */
   shouldApplyApplicationRole?: boolean;
+
+  /** The minimum mouse move distance to treat it as drag event */
+  distanceSquaredForDrag?: number;
```

Comment:
If we go with this approach, I'd have the user not pass in the squared distance, but just the distance and to any calculation detail ourselves. this leaks an implementation detail. 

We should probably also include that it's pixels? Maybe at least on the comment